README.md

```diff
@@ -1 +1,33 @@
-# HH-btag
\ No newline at end of file
+# HH-btag   
+Interface providing feature computation for CMS RunII HH-btag  NN to the identification of the reconstructed b-jets for the final state of hh->bbtautau
+
+# Installation
+
+1. cmsrel CMSSW_10_2_20
+1. cd CMSSW_10_2_20/src
+1. cmsenv
+1. git clone git@github.com:hh-italian-group/HHbtag.git
+1. scram b
+
+# Testing
+1. hhBtagTest
+Will produce the resulting scores for the input values for two events
+
+# Features
+
+The features used for the training are:
+
```

Comment:
You need to specify
- that jets must be selected using bbtautau baseline selection for the b jet candidates and reference the twiki.
- max number of jets that will be considered = 10
- jets must be ordered by DeepJet score in descending order